src/globus_cli/commands/flows/__init__.py

```diff
@@ -9,6 +9,8 @@
         "list": (".list", "list_command"),
         "show": (".show", "show_command"),
         "start": (".start", "start_command"),
+        # "runs" is a subgroup of commands.
+        "runs": (".runs", "runs_command"),
```

Comment:
I don't have a strong opinion, but I think we should discuss `runs` vs `run` as the name here.

`flows` is plural because the name of the service is Globus Flows -> `globus flows ...`
But a `run` is a singular object -> `run delete`

We already have several singulars like `globus task` (Transfer Task), `globus search task`, etc.

We also have _one_ case (as far as I know) where we made a mistake and have a singular for a service name which is supposed to be pluralized: Globus Groups -> `globus group`. (oops!)

What I'm saying is:
- there is some prior art which suggests a singular for an object name, and a plural/singular choice to match the service name
- but we haven't necessarily applied that set of rules consistently
- it's therefore unclear what the best choice is for `run` vs `runs`

I think I tend towards `globus flows run` (singular) and treating `globus group ...` as a minor bug which we should fix at some point by aliasing to `globus groups ...`. But I could be convinced otherwise.

Reply:
I'm on board for `run` (singular). I had thought I saw a "permissions" directory -- plural -- and followed that without double-checking that initial glance. So! Singular it is.

Reply:
> I'm on board for `run` (singular). I had thought I saw a "permissions" directory -- plural -- and followed that without double-checking that initial glance. So! Singular it is.

FWIW, I'm pretty sure you saw correctly and it's another place which is inconsistent with the other cases. (At what point are the inconsistent things so numerous that we can't talk confidently about consistency? :sweat_smile: )

I'd like to put together a game plan for aliasing and deprecating old names, but I haven't written anything down yet.

Reply:
As a general rule (if we're defining the norm), idk how I feel about `globus <service> <domain obj> <verb>` as a cli structure

I think `globus <service> <operation>` makes more sense (so `globus flows delete_run`)

Reply:
I guess our current flows approach is to treat the `domain obj` as implied if the object name matches the cli structure.

So `globus flows show` shows a flow because `flows` implies `globus flows flow show` 

But that pattern feels decently wrong to me.

Reply:
Eh, I'm not sure whether this PR is the right place to have this discussion.

I guess it does add a new sub-keyword to the CLI so maybe it is

Reply:
Discussed offline w/ Kurt & Stephen.

We're going to go with `globus flows run delete` for internal consistency in the cli (even if the CLI is not entirely consistent, discrepencies are considered the exception not the rule for this version)

We'll evaluate the naming structure of `globus <service> <operation>` as a potential migration in a subsequent major version (with the idea of maintaining aliasing for backwards compatability)